receiver/splunkenterprisereceiver/factory.go

```diff
@@ -2,3 +2,63 @@
 // SPDX-License-Identifier: Apache-2.0
 
 package splunkenterprisereceiver // import "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/splunkenterprisereceiver"
+
+import (
+	"context"
+	"time"
+
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/consumer"
+	"go.opentelemetry.io/collector/receiver"
+	"go.opentelemetry.io/collector/receiver/scraperhelper"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/splunkenterprisereceiver/internal/metadata"
+)
+
+const (
+	defaultInterval          = 10 * time.Minute
+	defaultMaxSearchWaitTime = 60 * time.Second
+)
+
+func createDefaultConfig() component.Config {
+	scfg := scraperhelper.NewDefaultScraperControllerSettings(metadata.Type)
+	scfg.CollectionInterval = defaultInterval
+
+	return &Config{
+		ScraperControllerSettings: scfg,
+		MetricsBuilderConfig:      metadata.DefaultMetricsBuilderConfig(),
+		MaxSearchWaitTime:         defaultMaxSearchWaitTime,
```

Comment:
`MaxSearchWaitTime` should not be needed. `ScraperControllerSettings` has `Timeout` config parameter for this purpose. Please use that one instead.

Reply:
Heard, thanks. Missed that in the ScraperControllerSettings sdk!

Reply:
Actually, can i include this change in a subsequent PR when i refactor the component to use configauth/basicauthextension? This is an easy change to implement in the component itself but I think it would break tests and I would rather only have to rewrite those one time